sdk/program/src/alt_bn128/compression.rs

```diff
@@ -51,13 +51,14 @@ impl From<u64> for AltBn128CompressionError {
 
 impl From<AltBn128CompressionError> for u64 {
     fn from(v: AltBn128CompressionError) -> u64 {
+        // note: should never return 0, as it risks to be confused with syscall success
```

Comment:
I think we can remove this `impl` all together, since it's not being used now? We can also most likely remove the `AltBn128CompressionError` itself too - the syscall is always going to return `Err(1)`, so there is no way we can get distinct error variants when calling functions/methods from `solana_program::alt_bn128` in Solana program context.

Reply:
Removed the from u64. This is still in use, though under feature gate.